brigade.js

```diff
@@ -326,11 +71,6 @@ function run(e, p, job) {
 events.on("check_suite:requested", runSuite);
```

Comment:
Should we keep the issue comment checks?  Then, while we're in a hybrid state (v1 running integration tests, the rest running via v2), `/brig run|check` would trigger re-running these as well, potentially.

Reply:
Here yes. Perhaps we should. Fixing.

Reply:
Fixed.